borg/testsuite/archiver.py

```diff
@@ -1298,6 +1298,29 @@ def test_fuse(self):
     def test_debug_put_get_delete_obj(self):
         pass
 
+    def test_strip_components_doesnt_leak(self):
```

Comment:
Separate this change (--strip-components leaking, test+fix) from the preload changes?